App7/App7/7(1).cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+internal class Program
+{
+    static void Main(string[] args)
+    {
+        var a = GetNumber("a");
+        var b = GetNumber("b");
```

Comment:
В задании числа называются по-другому. Переменные можете называть, как вам удобнее, но для пользователя вывод на консоль должен соответствовать заданию, а не вводить в заблуждение.
```suggestion
        var a = GetNumber("m");
        var b = GetNumber("n");
```